install.sh

```diff
@@ -1262,123 +916,14 @@ VMM_EXEC_KVM_SCRIPTS="src/vmm_mad/remotes/kvm/cancel \
                     src/vmm_mad/remotes/kvm/resize \
                     src/vmm_mad/remotes/kvm/resize_disk"
 
+VMM_EXEC_KVM_LIB="src/vmm_mad/remotes/lib/kvm/opennebula_vm.rb"
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

VMM_EXEC_KVM_LIB appears unused. Verify use (or export if used externally).